python/lsst/meas/base/forcedPhotCcd.py

```diff
@@ -778,18 +1037,43 @@ def __init__(self, butler=None, refSchema=None, initInputs=None, **kwds):
 
     def runQuantum(self, butlerQC, inputRefs, outputRefs):
         inputs = butlerQC.get(inputRefs)
+
+        tract = butlerQC.quantum.dataId['tract']
+        skyMap = inputs.pop('skyMap')
+        inputs['refWcs'] = skyMap[tract].getWcs()
```

Comment:
So sorry for not noticing this the first round. 

This is more wrong than what was here before. The `refWcs` is not the tract WCS and adding `skymap` as an input to this task is unnecessary.  This Task only runs on data frames with ra/decl coords. No WCS needed  nor allowed.  See validation message in `self.validate`.  

Remove your change, and below where it used to say:
```
inputs['refWcs'] = inputs['exposure'].getWcs()
```

you're welcome to change it to:
```
inputs['refWcs'] = None
```

which is probably clearer than what was there before.

Reply:
After wading through the code, I see now that `generateMeasCat` takes in a `refWcs` argument that is never used by anything at all (???), and `attachFootprints` only uses `refWcs` when used in `transformed` mode which isn't allowed for this task.  So I'll change it to `None` which is indeed clearer than what was there before.